include/openPMD/IO/AbstractIOHandlerImpl.hpp

```diff
@@ -117,26 +117,26 @@ class AbstractIOHandlerImpl
 
   /** Create a new file in physical storage, possibly overriding an existing file.
    *
-   * The operation should fail if m_handler->accessType is AccessType::READ_ONLY.
```

Comment:
The documentation here has been outdated since the day we started distinguishing `accessTypeBackend` from `accessTypeFrontend`. Maybe now is a good time to fix that?